android/sdk/src/main/java/org/jitsi/meet/sdk/NotificationUtils.java

```diff
@@ -0,0 +1,111 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.meet.sdk;
+
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.support.v4.app.NotificationCompat;
+import android.util.Log;
+
+
+class NotificationUtils {
+    private static final String TAG = NotificationUtils.class.getSimpleName();
+
+    private static final String ONGOING_CONFERENCEE_CHANNEL_ID = "JitsiNotificationChannel";
+    private static final String ONGOING_CONFERENCEE_CHANNEL_NAME = "Ongoing Conference Notifications";
+
+    static void createOngoingConferenceNotificationChannel() {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
+            return;
+        }
+
+        Context context = ReactInstanceManagerHolder.getCurrentActivity();
+        if (context == null) {
+            Log.w(TAG, "Cannot create notification channel: no current context");
+            return;
+        }
+
+        NotificationManager notificationManager
+            = (NotificationManager)context.getSystemService(Context.NOTIFICATION_SERVICE);
+
+        NotificationChannel channel
+            = notificationManager.getNotificationChannel(ONGOING_CONFERENCEE_CHANNEL_ID);
+        if (channel != null) {
+            // The channel was already created, no need to do it again.
+            return;
+        }
+
+        channel = new NotificationChannel(
+                ONGOING_CONFERENCEE_CHANNEL_ID,
+                ONGOING_CONFERENCEE_CHANNEL_NAME,
+                NotificationManager.IMPORTANCE_DEFAULT);
+        channel.enableLights(false);
+        channel.enableVibration(false);
+        channel.setShowBadge(false);
+
+        notificationManager.createNotificationChannel(channel);
+    }
+
+    static Notification buildOngoingConferenceNotification() {
+        Context context = ReactInstanceManagerHolder.getCurrentActivity();
+        if (context == null) {
+            Log.w(TAG, "Cannot create notification: no current context");
+            return null;
+        }
+
+        Intent notificationIntent = new Intent(context, context.getClass());
+        PendingIntent pendingIntent = PendingIntent.getActivity(context, 0, notificationIntent, 0);
+
+        NotificationCompat.Builder builder;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            builder = new NotificationCompat.Builder(context, ONGOING_CONFERENCEE_CHANNEL_ID);
+        } else {
+            builder = new NotificationCompat.Builder(context);
+        }
+
+        builder
+            .setCategory(NotificationCompat.CATEGORY_CALL)
+            .setContentTitle("Ongoing meeting")
```

Comment:
Would you consider moving these strings to strings.xml resource file

Reply:
Sure.